shared/locales/de/website-partners.json

```diff
@@ -28,284 +28,298 @@
 		"founded": "Gründung",
 		"partner-since": "Partner seit",
 		"headquarter": "Hauptsitz",
-		"mission": "Mission"
+		"mission": "Mission",
+		"fundraiser": "Fundraiser",
```

Comment:
@ssandino 
Could you help with the Fundraiser translation in de, it and fr?

Reply:
German:
	•	Mission: Mission
	•	Fundraiser: Spendenaktion

Italian:
	•	Mission: Missione
	•	Fundraiser: Raccolta fondi

French:
	•	Mission: Mission
	•	Fundraiser: Collecte de fonds